test/activity_test.go

```diff
@@ -87,19 +87,12 @@ func (a *Activities) fail(_ context.Context) error {
 	return errFailOnPurpose
 }
 
-func (a *Activities) InspectActivityInfo(ctx context.Context, domain, taskList, wfType string) error {
-	a.append("inspectActivityInfo")
-	info := activity.GetInfo(ctx)
-	if info.WorkflowDomain != domain {
-		return fmt.Errorf("expected domainName %v but got %v", domain, info.WorkflowDomain)
-	}
-	if info.WorkflowType == nil || info.WorkflowType.Name != wfType {
-		return fmt.Errorf("expected workflowType %v but got %v", wfType, info.WorkflowType)
-	}
-	if info.TaskList != taskList {
-		return fmt.Errorf("expected taskList %v but got %v", taskList, info.TaskList)
+func (a *Activities) DuplicateStringInContext(ctx context.Context) (string, error) {
+	originalString := ctx.Value(contextKey(testContextKey))
+	if originalString == nil {
+		return "", fmt.Errorf("context did not propagate to local activity")
```

Comment:
nit: remove "local" in error message, as the activity implementation itself is agnostic of whether it is local or not.

Reply:
got it